internal/cli/universal_login_customize.go

```diff
@@ -342,16 +343,21 @@ func (h *webSocketHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	connection, err := upgrader.Upgrade(w, r, nil)
 	if err != nil {
-		h.display.Errorf("failed to upgrade the connection to the WebSocket protocol: %v", err)
+		h.display.Errorf("Failed to upgrade the connection to the WebSocket protocol: %v", err)
+		h.display.Warnf("Try restarting the command.")
 		h.shutdown()
 		return
 	}
+	defer func() {
+		_ = connection.Close()
```

Comment:
We weren't properly closing the connection before. This is really important to do.